catalog-info.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: backstage.io/v1alpha1
+kind: Component
+metadata:
+    name: Newa
+    description: |
+      New Errata Workflow Automation.
+      Automation tool for automated testing of Red Hat Erratums using tmt, Testing Farm, ReportPortal and Jira.
+
+    # https://backstage.io/docs/features/software-catalog/descriptor-format#links-optional
+    links:
+        - title: user guide
+          url: https://docs.google.com/document/d/1kRBXCGVopImAJYX7KMZA0RJFhW2a_U03yYFJ_2Bg-eA/edit?usp=sharing
```

Comment:
Is it wise to share URL of an internal document? Should I expect access requests from public?

Reply:
No, this file is for internal catalog, I would put it here :)

Reply:
But the link would be in a public repo.

Reply:
yes, I do not see an issue with that, but a good discussion point :)

Reply:
docs.google.com is no secret is it?

This file will be consumed only by internal RH users. We will replace once there is a nice user guide somewhere here and public?

Reply:
Added to next meeting agenda :)